clean/houma_pd_cprr.py

```diff
@@ -1,135 +0,0 @@
-    df.loc[:, 'disposition'] = df.inv_disposition.str.lower().str.strip()\
-        .str.replace(r'^sust[aä]n?ined ?\/? ?(.+)', r'sustained', regex=True)\
-        .str.replace(r'(all officers |equipment|verbal counseling)', '', regex=True)\
-        .str.replace(r'^written (.+)', '', regex=True)\
-        .str.replace(r'(.+)? ?(suspension) ?(.+)?', '', regex=True)
-    return df.drop(columns='inv_disposition')
-
-
-def split_and_clean_investigator(df):
-    investigators = df.investigator_column6_column5_column4_column3_column2_columnt.str.lower().str.strip()\
-        .str.replace(r'Sgt\.Dave Wagner ', 'sgt david wagner', regex=True)\
-        .str.replace('jarrad', 'jarrod', regex=False)\
-        .str.replace('magtherne', 'matherne', regex=False)\
-        .str.extract(r'(sgt)\.? (\w+) (\w+)')
-
-    df.loc[:, 'investigator_rank_desc'] = investigators[0]\
-        .str.replace('sgt', 'sergeant', regex=False)
-    df.loc[:, 'investigator_first_name'] = investigators[1]
-    df.loc[:, 'investigator_last_name'] = investigators[2]
-    return df.drop(columns='investigator_column6_column5_column4_column3_column2_columnt')
-
-
-def clean_ia_tracking_number(df):
-    df.loc[:, 'external_tracking_number'] = df.pd_case_number.str.lower().str.strip()\
-        .str.replace(r'pd(.+)', r'pd-\1', regex=True)\
-        .str.replace(r'lsp(.+)', r'lsp-\1', regex=True)
-    return df.drop(columns='pd_case_number')
-
-
-def split_and_clean_names(df):
-    df.loc[:, 'officer'] = df.officer.str.lower().str.strip()\
-        .str.replace('sidneytherlot', 'sidney therlot', regex=False)\
-        .str.replace(r'(polk street ois|houma police dept)', '', regex=True)
-
-    names = df.officer.str.extract(r'(\w+) (\w+)')
-
-    df.loc[:, 'first_name'] = names[0].fillna('')
-    df.loc[:, 'last_name'] = names[1].fillna('')
-    return df.drop(columns='officer')
-
-
-def drop_rows_missing_first_and_last_name(df):
-    return df[~((df.first_name == '') & (df.last_name == ''))]
-
-
-def clean():
-    df = pd.read_csv(data_file_path('raw/houma_pd/houma_pd_cprr_2019_2021.csv'))\
-        .pipe(clean_column_names)\
-        .rename(columns={
-            'case_number': 'tracking_number'
-        })\
-        .pipe(split_rows_with_multiple_allegations)\
-        .pipe(clean_allegations)\
-        .pipe(clean_ia_tracking_number)\
-        .pipe(extract_action)\
-        .pipe(clean_disposition)\
-        .pipe(split_and_clean_investigator)\
-        .pipe(split_and_clean_names)\
-        .pipe(drop_rows_missing_first_and_last_name)\
-        .pipe(standardize_desc_cols, ['tracking_number', 'action'])\
-        .pipe(set_values, ({'agency': 'Houma PD'}))\
-        .pipe(gen_uid, ['agency', 'first_name', 'last_name'])\
-        .pipe(gen_uid, ['uid', 'tracking_number', 'external_tracking_number', 'allegation', 'action'], 'allegation_uid')
-    return df
-
-
-if __name__ == '__main__':
-    df = clean()
-    df.to_csv(data_file_path('clean/cprr_houma_pd_2019_2021.csv'))
```

Comment:
Do you need to remove this file?

Reply:
I intended to commit and push a new branch for houma_pd_cprr_2019_2021 after reverting main to its state prior to the introduction of houma_pd_cprr.